test/regression/api/admin/members.test.js

```diff
@@ -4,34 +4,36 @@ const should = require('should');
 const supertest = require('supertest');
 const sinon = require('sinon');
 const testUtils = require('../../../utils');
+const framework = require('../../../utils/e2e-framework');
 const localUtils = require('./utils');
 const config = require('../../../../core/shared/config');
 const labs = require('../../../../core/shared/labs');
 const mailService = require('../../../../core/server/services/mail');
 
-let request;
+let agent;
+let emailStub;
 
 describe('Members API', function () {
     before(function () {
         sinon.stub(labs, 'isSet').withArgs('members').returns(true);
     });
 
-    after(function () {
-        sinon.restore();
+    after(async function () {
+        await framework.resetDb();
     });
 
     before(async function () {
-        await localUtils.startGhost();
-        request = supertest.agent(config.get('url'));
-        await localUtils.doAuth(request, 'members');
+        agent = await framework.getAgent('/ghost/api/canary/admin/');
+        await framework.initFixtures('members');
+        await agent.loginAsOwner();
     });
 
     beforeEach(function () {
-        sinon.stub(mailService.GhostMailer.prototype, 'send').resolves('Mail is disabled');
+        emailStub = framework.stubMail();
```

Comment:
I guess we'll need to setup this to be able to stub the Stripe API - but I think that'll be straightforward